### ISSUE TITLE
Kubernetes dial tcp 127.0.0.1:80: connect: connection refused && Kubernetes cluster unreachable: invalid configuration: no configuration has been provided, try setting KUBERNETES_MASTER environment variable

### ISSUE THREAD

**Issue by 0x57696c6c** [SygniaLabs/ScallOps#24]
Using branch v0,4 as the master branch is returning even more errors during the terraform build (terraform v1.2.6).
I'm starting with a brand new GCP project, and enabled GCE, k8s, storage, containers, and secrets.
Configured config.tfvars to include my project_id, infra_name, and backups_bucket_name,
Authenticated using the gcloud cli.
Initialized terraform and ran terraform apply --var-file=./config.tfvars.
Service accounts are created, the gitlab vm is started, the k8s cluster is created with nodes, storage bucket created, firewall rules in place.
However, towards the end of the terraform build I constantly receive the following errors:
```
Error: Get "http://localhost/api/v1/namespaces/default/secrets/newerscallops-gitlab.us-central1-a.c.newer-scallops.internal-cert": dial tcp 127.0.0.1:80: connect: connection refused
│
│   with kubernetes_secret.k8s_gitlab_cert_secret,
│   on gke.tf line 6, in resource "kubernetes_secret" "k8s_gitlab_cert_secret":
│    6: resource "kubernetes_secret" "k8s_gitlab_cert_secret" {
│
╵
╷
│ Error: Get "http://localhost/api/v1/namespaces/sensitive": dial tcp 127.0.0.1:80: connect: connection refused
│
│   with kubernetes_namespace.sensitive-namespace,
│   on gke.tf line 18, in resource "kubernetes_namespace" "sensitive-namespace":
│   18: resource "kubernetes_namespace" "sensitive-namespace" {
│
╵
╷
│ Error: Get "http://localhost/apis/policy/v1beta1/namespaces/kube-system/poddisruptionbudgets/k8s-pdb-kube-dns": dial tcp 127.0.0.1:80: connect: connection refused
│
│   with kubernetes_pod_disruption_budget.kube-dns,
│   on gke.tf line 223, in resource "kubernetes_pod_disruption_budget" "kube-dns":
│  223: resource "kubernetes_pod_disruption_budget" "kube-dns" {
│
╵
╷
│ Error: Get "http://localhost/apis/policy/v1beta1/namespaces/kube-system/poddisruptionbudgets/k8s-pdb-konnectivity-agent": dial tcp 127.0.0.1:80: connect: connection refused
│
│   with kubernetes_pod_disruption_budget.konnectivity-agent,
│   on gke.tf line 240, in resource "kubernetes_pod_disruption_budget" "konnectivity-agent":
│  240: resource "kubernetes_pod_disruption_budget" "konnectivity-agent" {
│
╵
╷
│ Error: Kubernetes cluster unreachable: invalid configuration: no configuration has been provided, try setting KUBERNETES_MASTER environment variable
│
│   with helm_release.gitlab-runner-linux,
│   on main.tf line 68, in resource "helm_release" "gitlab-runner-linux":
│   68: resource "helm_release" "gitlab-runner-linux" {
│
╵
╷
│ Error: Kubernetes cluster unreachable: invalid configuration: no configuration has been provided, try setting KUBERNETES_MASTER environment variable
│
│   with helm_release.gitlab-runner-win,
│   on main.tf line 172, in resource "helm_release" "gitlab-runner-win":
│  172: resource "helm_release" "gitlab-runner-win" {
```

RE: #23  - Sorry for the double post, but I was able to continue the terraform build by adding the _<project_id>-gke-buckt_ service account to IAM with the storage admin role. Although I am not certain this actually resolves #23.

**Comment by gilb1** [comment]
Hi there!
Thanks for checking out the deployment and detailing issues, somehow I didn't receive any notification about your issues.
We are in the process of merging the next version which is still under testing.  We'll take your issues into consideration.
For this specific issue, there were few commits released (to 0.4) that resolve it, as we also experienced the same.
Next time you attempt to deploy, feel free to ping me via Discord and I'll join you to see whether there are additional issues.
Discord: B1t0n#4141
Twitter: B1t0n_
@0x57696c6c

**Comment by gilb1** [comment]
@0x57696c6c  Hi, can you give the deployment another try and report success/fail so I we'll know to close the issue
